iothub_client/tests/iothubclient_ut/iothubclient_ut.c

```diff
@@ -2487,6 +2497,168 @@ TEST_FUNCTION(IoTHubClient_UploadToBlobAsync_succeeds)
     
     IoTHubClient_Destroy(iothub_handle);
 }
+
+/*SRS_IOTHUBCLIENT_99_072: [ If `iotHubClientHandle` is `NULL` then `IoTHubClient_UploadMultipleBlocksToBlobAsync` shall fail and return `IOTHUB_CLIENT_INVALID_ARG`. ]*/
```

Comment:
Some of the requirement IDs are written without the "Tests_" prefix (to make them /* Tests_SRS_...)
Sorry, a little burden, I know, but that is part of an automation we have to check for requirement strings between code and tests.

Reply:
I should have seen this too! Btw, do you plan to release this kind automation tool?